src/utils.ts

```diff
@@ -107,3 +107,27 @@ export function truncateAptSuffix(name: string): string {
     "$1",
   );
 }
+
+/**
+ * Standardizes an address to the format "0x" followed by 64 lowercase hexadecimal digits.
+ */
+export const standardizeAddress = (address: string): string => {
```

Comment:
Is this consistent with the standardize address in the indexer? 

/// Standardizes all addresses and table handles to be length 66 (0x-64 length hash)
pub fn standardize_address(handle: &str) -> String {
    format!("0x{:0>64}", &handle[2..])
}

Reply:
yeah i believe so. can i get additional eyes on this @bowenyang007 @larry-aptos  to make sure it aligns with indexer way of standardizing address? [example](https://github.com/aptos-labs/aptos-core/blob/e149dc164f1ea77d410ac8fc29780b17cc7ec778/crates/indexer/src/util.rs#L15)

Reply:
looks good. the code jill referring to takes invalid cases into consideration.